packages/showcase/src/user-controls/common-snippets/common-snippets.component.html

```diff
@@ -17,6 +17,7 @@
   }
 
   @case ("spacing-body-components") {
+    <!-- Include here a component if it adheres to the spacing body classes -->
```

Comment:
Avoid adding HTML comment in a JS framework, since it can lead to render issues.

```suggestion
```